frontend/src/components/FacilityBooking/ViewBlock.tsx

```diff
@@ -19,15 +22,24 @@ const ViewBlock = (props: Props) => {
   }, [ref.current])
 
   return (
-    <StyledViewBooking ref={ref} isOccupied={props.entry.type === TimeBlockType.OCCUPIED} blockId={props.entry.id}>
-      {props.entry.type === TimeBlockType.OCCUPIED && (
-        <>
-          {props.entry.ccaName}
-          <br />
-          {props.entry.eventName}
-        </>
+    <>
+      {props.entry.type === TimeBlockType.OCCUPIED ? (
+        <StyledViewBooking
+          ref={ref}
+          isOccupied={true}
```

Comment:
this should be sufficient
```suggestion
          isOccupied
```

Reply:
Fixed in 38fbead3d